server/lib/field_hub_web/live/project_show_live.ex

```diff
@@ -112,6 +117,75 @@ defmodule FieldHubWeb.ProjectShowLive do
     {:noreply, assign(socket, :new_password, password)}
   end
 
+  # def handle_event("delete_cache", %{"project" => project} = _values, socket) do
+  def handle_event("delete_cache", _values, %{assigns: %{project: project}}=socket) do
+    new_cache = "Delete or update cache in this handle_event"
+    cache_index = FieldHub.FileStore.file_index(project)
```

Comment:
For the ticket you do not need to create a new cache, that is already handled by the `FileStore` internally.

The only thing that has to be triggered from the interface is the FileStore's function to clear the existing cache. If you want to have a look how the file index looks like, you could for example use the Desktop client to push new images into the repository.